app/guid-file/template.hbs

```diff
@@ -67,137 +67,156 @@
-                                </Button>
+                                <div local-class='metadata-form-and-buttons'>
+                                    <div local-class='edit-metadata-form'>
+                                        <FormControls @changeset={{manager.changeset}} as |form|>
+                                            <form.textarea
+                                                data-test-title-field
+                                                @valuePath='title'
+                                                @label={{t 'file_detail.title'}}
+                                            />
+                                            <form.textarea
+                                                data-test-description-field
+                                                @valuePath='description'
+                                                @label={{t 'file_detail.description'}}
+                                            />
+                                            <form.select
+                                                data-test-select-resource-type
+                                                @options={{manager.resourceTypeGeneralOptions}}
+                                                @label={{t 'file_detail.resource_type_general'}}
+                                                @valuePath='resourceTypeGeneral'
+                                                as |option|
+                                            >
+                                                {{option}}
+                                            </form.select>
+                                            <form.select
+                                                data-test-select-resource-language
+                                                @options={{manager.languageCodes}}
+                                                @label={{t 'file_detail.language'}}
+                                                @onchange={{manager.changeLanguage}}
+                                                @valuePath='languageObject'
+                                                as |option|
+                                            >
+                                                {{option.name}}
+                                            </form.select>
+                                        </FormControls>
+                                    </div>
+                                    <div local-class='edit-metadata-save-cancel-buttons'>
+                                        <Button
+                                            data-test-cancel-editing-metadata-button
+                                            aria-label={{t 'general.cancel'}}
+                                            data-analytics-name='Cancel editing metadata'
+                                            @layout='fake-link'
+                                            local-class='cancel-edit-button'
```

Comment:
This button here can be simplified to use the default secondary button styling with
```suggestion
                                            @type='secondary'
```